### PR TITLE
Update privacy policy with hoc cookie notice

### PR DESCRIPTION
**Pull request by dju90**
to include a link to HourofCode.com/cookies

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dju90** [comment]
Not sure why github is flagging lines 331 or 333 as a change, as that wasn't changed/doesn't appear to be a change in the diff

File: pegasus/sites.v3/code.org/public/privacy.md [hunk 1]
```diff
@@ -330,4 +330,4 @@ Where applicable, we will not make any material changes to the Privacy Policy th
 
 <a name="contacting-codeorg"></a>
 ## Contacting Code.org
-Please contact Code.org with any questions or comments about this Privacy Policy by emailing us at privacy@code.org or writing to us at Code.org (Attn. Privacy), 1501 Fourth Ave, Suite 900, Seattle, WA 98101. As set forth above, requests to access, update, or delete your personal data should be submitted by email to [support@code.org](mailto:support@code.org) or entered at: [https://code.org/contact](https://code.org/contact).
\ No newline at end of file
+Please contact Code.org with any questions or comments about this Privacy Policy by emailing us at privacy@code.org or writing to us at Code.org (Attn. Privacy), 1501 Fourth Ave, Suite 900, Seattle, WA 98101. As set forth above, requests to access, update, or delete your personal data should be submitted by email to [support@code.org](mailto:support@code.org) or entered at: [https://code.org/contact](https://code.org/contact).
```
Comment (wilkie) [inline pegasus/sites.v3/code.org/public/privacy.md:333]: It bugs me that it thinks this is a change. Why does it think this is a change.

**Review by wilkie — APPROVED** [review]
Links are correct and work. 👍

**Review by mgc1194 — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/privacy.md (hunk 1 above)
Comment (dju90) [reply pegasus/sites.v3/code.org/public/privacy.md:333]: Hard same...I checked them in a string comparison tool and they are identical 🤷

**Pull request merged**